package.json

```diff
@@ -1,83 +1,76 @@
+    "test-watch": "cross-env NODE_ENV=test mocha -r esm --watch --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "cross-env NODE_ENV=test nyc report",
+    "lint": "eslint ."
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-web-local-vc-store",
   "author": {
     "name": "Digital Bazaar, Inc.",
     "email": "support@digitalbazaar.com",
     "url": "https://digitalbazaar.com/"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/digitalbazaar/isomorphic-js-lib-template"
+    "url": "https://github.com/digitalbazaar/bedrock-web-local-vc-store"
   },
   "bugs": {
-    "url": "https://github.com/digitalbazaar/isomorphic-js-lib-template/issues",
+    "url": "https://github.com/digitalbazaar/bedrock-web-local-vc-store/issues",
     "email": "support@digitalbazaar.com"
   },
-  "license": "BSD-3-Clause",
   "files": [
-    "lib/*.js"
+    "lib/"
   ],
-  "main": "lib/index.js",
-  "module": "lib/main.js",
   "dependencies": {
-    "esm": "^3.2.25"
+    "pouchdb": "^7.2.2",
+    "pouchdb-adapter-memory": "^7.2.2",
+    "pouchdb-find": "^7.2.2"
   },
+  "keywords": [
+    "Verifiable",
+    "Credential",
+    "Query",
+    "Linked Data"
+  ],
   "devDependencies": {
-    "@babel/core": "^7.13.8",
-    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
-    "@babel/plugin-transform-runtime": "^7.13.9",
-    "@babel/preset-env": "^7.13.9",
-    "@babel/runtime": "^7.13.9",
-    "babel-loader": "^8.2.2",
-    "chai": "^4.3.3",
-    "cross-env": "^7.0.3",
-    "eslint": "^7.21.0",
-    "eslint-config-digitalbazaar": "^2.6.1",
-    "eslint-plugin-jsdoc": "^32.2.0",
-    "karma": "^6.1.1",
-    "karma-babel-preprocessor": "^8.0.1",
+    "@babel/core": "^7.4.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.4.3",
+    "@babel/plugin-transform-runtime": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/runtime": "^7.4.3",
+    "babel-loader": "^8.0.5",
+    "bedrock-web-local-vc-store": "file:.",
+    "chai": "^4.3.4",
+    "cross-env": "^6.0.3",
+    "eslint": "^7.3.1",
+    "eslint-config-digitalbazaar": "^2.5.0",
+    "eslint-plugin-jsdoc": "^30.5.2",
+    "esm": "^3.2.22",
+    "karma": "^4.0.1",
+    "karma-babel-preprocessor": "^8.0.0",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^3.1.0",
-    "karma-mocha": "^2.0.1",
+    "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
-    "karma-sourcemap-loader": "^0.3.8",
-    "karma-webpack": "^5.0.0",
-    "mocha": "^8.3.1",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^4.0.2",
+    "mocha": "^7.0.0",
     "mocha-lcov-reporter": "^1.3.0",
-    "nyc": "^15.1.0",
-    "webpack": "^5.24.3"
+    "nyc": "^15.0.0",
+    "webpack": "^4.29.6"
```

Comment:
Update all the deps to at least the versions they were at before, if not newer.  It looks like they were downgraded from the template via copy & paste?